### PR TITLE
Add Router logic

### PR DESCRIPTION
**Pull request by ericdrosado**

### CODE REVIEW

File: Sources/Request/Request.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+         self.path = path
+         self.queries = queries
+     }
+
+     public init(method: String, path: String, cookie: String) {
+         self.method = method
+         self.path = path
+         self.cookie = cookie 
+     }
+
```
Comment (cdemyanovich) [inline Sources/Request/Request.swift]: We may have discussed these `init` methods before, but I want to touch on them again. They're all slightly different in the parts of the `Request` that they initialize. Is there a way to consolidate to one that specifies default values for the parts that might be absent? For example,

```swift
public init(method: String, path: String, body: [String: String] = [:], queries: [String: String] = [:], cookie: String) {
  ...
}
```

File: Sources/Routes/Root.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Request
+
+public class Root: Route {
+
+    public init(){}
+
+    public func handleRoute(request: Request) -> String {
+        var body = request.path == "/" ? "Hello World": String()
```
Comment (cdemyanovich) [inline Sources/Routes/Root.swift]: If we're in `Root`, don't we know that `request.path == "/"`?

**Review by cdemyanovich — COMMENTED** [review]
Why do I see "You are viewing a condensed version of this merge commit. You can view the full changes here." at the bottom of the "Files changed" tab when reviewing the PR?

File: Tests/Test/ResponseTest.swift [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import XCTest
+import Routes
 import Server
 
 class ResponseTest: XCTestCase {
     
     let parser = Parser()
-    let response = Response()
+    static let routes: [String: Route] = ["/": Root(), "/hello": Hello(), "/tea": Tea(), "/coffee": Coffee(), "/parameters": Parameters(), "/cookie": Cookie(), "/eat_cookie": EatCookie(), "/redirect": Redirect(), "/form": Form(), "/method_options": MethodOptions(), "/method_options2": MethodOptions2()]
```
Comment (tommcgee) [inline Tests/Test/ResponseTest.swift]: Seems like you may run into some issues keeping these routes up to date

Comment (ericdrosado) [reply Tests/Test/ResponseTest.swift]: Yes, I definitely "felt the pain" on that one. I plan on addressing that so I don't have to write these routes twice. Once for test and once for Main.

**Pull request merged**